ts/screens/onboarding/EmailScreen.tsx

```diff
@@ -0,0 +1,148 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { untag } from "italia-ts-commons/lib/types";
+import { Text, View } from "native-base";
+import * as React from "react";
+import { Alert, StyleSheet } from "react-native";
+import { NavigationScreenProps } from "react-navigation";
+import { connect } from "react-redux";
+
+import ScreenContent from "../../components/screens/ScreenContent";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import IconFont from "../../components/ui/IconFont";
+import Markdown from "../../components/ui/Markdown";
+import I18n from "../../i18n";
+import { BiometrySimpleType } from "../../sagas/startup/checkAcknowledgedFingerprintSaga";
+import {
+  abortOnboarding,
+  emailAcknowledged
+} from "../../store/actions/onboarding";
+import { Dispatch, ReduxProps } from "../../store/actions/types";
+import { GlobalState } from "../../store/reducers/types";
+import customVariables from "../../theme/variables";
+
+type NavigationParams = {
+  biometryType: BiometrySimpleType;
+};
+
+type OwnProps = ReduxProps &
+  ReturnType<typeof mapStateToProps> &
+  NavigationScreenProps<NavigationParams>;
+
+type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
+
+const styles = StyleSheet.create({
+  emailLabel: { fontSize: 14 },
+  emailWithIcon: {
+    flexDirection: "row",
+    justifyContent: "flex-start",
+    alignItems: "center"
+  },
+  spacerSmall: { height: 12 },
+  spacerLarge: { height: 24 },
+  email: {
+    fontWeight: customVariables.h1FontWeight,
+    color: customVariables.h1Color,
+    fontSize: 18,
+    marginLeft: 8
+  }
+});
+
+const unavailableAlert = () => Alert.alert(I18n.t("global.notImplemented"));
+
+/**
+ * A screen to show if the fingerprint is supported to the user.
+ */
+export class FingerprintScreen extends React.PureComponent<Props> {
+  private handleGoBack = () =>
+    Alert.alert(
+      I18n.t("onboarding.alert.title"),
+      I18n.t("onboarding.alert.description"),
+      [
+        {
+          text: I18n.t("global.buttons.cancel"),
+          style: "cancel"
+        },
+        {
+          text: I18n.t("global.buttons.exit"),
+          style: "default",
+          onPress: () => this.props.abortOnboarding()
+        }
+      ]
+    );
+
+  public render() {
+    const { potProfile } = this.props;
+
+    const profileData = potProfile
```

Comment:
```suggestion
    const profileEmail = potProfile.map(_ => untag(_.spid_email)).getOrElse("");
```
Could we avoid an object since we use only the email field?

Reply:
changed in 7ea173d9